podcasts/Onboarding/Plus/UpgradeLandingView.swift

```diff
@@ -296,10 +341,16 @@ struct UpgradeCard: View {
         .shadow(color: .black.opacity(0.1), radius: 3, x: 0, y: 2)
         .shadow(color: .black.opacity(0.1), radius: 0, x: 0, y: 0)
     }
+
+    @ViewBuilder
+    var termsAndConditions: some View {
+        let purchaseTerms = L10n.purchaseTerms("$", "$", "$", "$").components(separatedBy: "$")
+        Text(purchaseTerms[safe: 0] ?? "") + Text("[\(purchaseTerms[safe: 1] ?? "")](https://support.pocketcasts.com/article/privacy-policy/)").underline() + Text(purchaseTerms[safe: 2] ?? "") + Text("[\(purchaseTerms[safe: 3] ?? "")](https://support.pocketcasts.com/article/terms-of-use/)").underline()
```

Comment:
I cannot use a constant for the URL here, unfortunately. If I do that, the link doesn't work.

Reply:
Awesome, thanks! Done in d8bbfc8548940e7b2306ad9833b54cd63c7b152c